dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -2249,15 +2249,15 @@ BEGIN
     2421 "Litera devanagari aw"
     2422 "Litera devanagari ue"
     2423 "Litera devanagari uue"
-    2424 "Devanagari Letter Marwari Dda"
+    2424 "Litera devanagari marwari dda"
     2425 "Litera devanagari zha"
     2426 "Litera devanagari ya greu"
```

Comment:
Since you've chosen to use "îngroșat", it's only right to address the rest of the cases (but I think it's only here) where the English "heavy" was left translated as "greu".

Reply:
Actually, I see that you haven't touched this. Please properly accord the noun "litera" with its "greu" adrective.

Reply:
Sorry, I forgot about this.